tests/losses/test_multi_scale.py

```
@@ -76,10 +76,10 @@ def test_ill_opts(self, kwargs, input, target):
             with self.assertRaisesRegex(ValueError, ""):
                 MultiScaleLoss(**kwargs)(input, target)
 
-    def test_script(self):
+    def test_export(self):
         input_param, input_data, expected_val = TEST_CASES[0]
```

Ruff (0.15.6)

[warning] Unpacked variable `expected_val` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)